lib/cli/driver.js

```diff
@@ -115,8 +120,14 @@ class DriverCommand {
       const updateTxt = showUpdates && updateVersion ?
         ` [${updateVersion} available]`.magenta :
         '';
+      const upToDateTxt = showUpdates && upToDate ?
+        ` [Up to date]`.green :
+        '';
+      const unsafeUpdateTxt = showUpdates && unsafeUpdateVersion ?
+        ` [${unsafeUpdateVersion} available]`.red :
```

Comment:
red color usually means some errors. Also, does it make sense to change the text message itself, so it differs from a `safe` one?

Reply:
i was going to use the red as an alert that there was an unsafe update available. i could change the color and add more language instead, just didn't want it to be too verbose.